### PR TITLE
PR: Fixes content type attribute error for global config dialog

### LINKED ISSUE #9691: Can't set other Python interpreter
### What steps will reproduce the problem?

- Open spyder;
- Try to set an other interpreter;

The error occurs when setting the path manualy. When I try to set it browsing the files spyder freeze.

### Traceback
```python-traceback
  File "/usr/lib/python3.7/site-packages/spyder/preferences/configdialog.py", line 222, in button_clicked
    configpage.apply_changes()
  File "/usr/lib/python3.7/site-packages/spyder/preferences/configdialog.py", line 96, in apply_changes
    self.save_to_conf()
  File "/usr/lib/python3.7/site-packages/spyder/preferences/configdialog.py", line 425, in save_to_conf
    if lineedit.content_type == list:
AttributeError: 'QLineEdit' object has no attribute 'content_type'
```

## Versions

* Spyder version: 4.0.0.dev0 
* Python version: 3.7.3
* Qt version: 5.12.3
* PyQt5 version: 5.12.2
* Operating System: Linux 4.19.49-1-MANJARO

### Dependencies

```
pygments >=2.0    :  2.4.2 (OK)
qdarkstyle >=2.6.4:  2.7 (OK)
sphinx >=0.6.6    :  2.1.0 (OK)
psutil >=0.3      :  5.6.2 (OK)
pyls >=0.27.0     :  0.26.1 (NOK)
nbconvert >=4.0   :  5.5.0.dev0 (OK)
pandas >=0.13.1   :  None (NOK)
numpy >=1.7       :  None (NOK)
sympy >=0.7.3     :  None (NOK)
cython >=0.21     :  None (NOK)
qtconsole >=4.5.0 :  4.5.1 (OK)
IPython >=4.0     :  7.5.0 (OK)
matplotlib >=2.0.0:  None (NOK)
pylint >=0.25     :  2.3.1 (OK)
```

### PR DESCRIPTION
**Pull request by goanpeca**
## Description of Changes

* [x] Wrote at least one-line docstrings (for any new functions)
* [x] Added unit test(s) covering the changes (if testable)

### Issue(s) Resolved

Fixes #9691

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct: @goanpeca

### CODE REVIEW

File: spyder/preferences/configdialog.py [hunk 1]
```diff
@@ -392,10 +392,13 @@ def load_from_conf(self):
              ), (option, default) in list(self.scedits.items()):
             edit = clayout.lineedit
             btn = clayout.colorbtn
-            color, bold, italic = self.get_option(option, default)
-            edit.setText(color)
-            cb_bold.setChecked(bold)
-            cb_italic.setChecked(italic)
+            options = self.get_option(option, default)
+            if options:
```
Comment (goanpeca) [inline spyder/preferences/configdialog.py]: @ccordoba12 hmmm I tried locally on a clean environment, not sure why `self.get_option(option, default)` would yield `None`.

Thougths?

Comment (ccordoba12) [reply spyder/preferences/configdialog.py]: What's the `option` and `default` values that are giving None?

Comment (goanpeca) [reply spyder/preferences/configdialog.py]: It has to be related to something here I guess https://github.com/spyder-ide/spyder/blob/master/spyder/preferences/appearance.py#L574 but it happens during testing only on CI (I cannot reproduce locally)

**Comment by goanpeca** [comment]
This one is ready @ccordoba12 (besides the weird error 🤷‍♂ that happened on CI before I added a "fix")

**Pull request merged**